sdk/servicebus/azure-messaging-servicebus/src/test/java/com/azure/messaging/servicebus/implementation/ManagementChannelTests.java

```diff
@@ -117,11 +137,9 @@ void teardown(TestInfo testInfo) {
     void setsSessionState(byte[] state) {
         // Arrange
         final String sessionId = "A session-id";
-        managementChannel = new ManagementChannel(Mono.just(requestResponseChannel), NAMESPACE, ENTITY_PATH, sessionId,
-            tokenManager, messageSerializer, Duration.ofSeconds(10));
 
         // Act
-        StepVerifier.create(managementChannel.setSessionState(state))
+        StepVerifier.create(managementChannel.setSessionState(sessionId, state, LINK_NAME))
```

Comment:
Another error case : Where user pass in a session id which is not equal to current active session id.

Reply:
This is done on the management channel. This id doesn't have to be the active one. It could be one that was passed in from another application that is also renewing the lock.

Reply:
This is done on the managememt channel. If another application passes you this id, you can complete it as long as someone is renewing that lock. the .NET code doesn't check that the session is not lost.

Reply:
If this is done on the actual receive link, then it checks to see if it is not lost.